lib/node.js

```diff
@@ -551,6 +551,15 @@ declare module "fs" {
 }
 
 declare module "http" {
+  declare class Server extends events$EventEmitter {
+    listen(port: number, hostname?: string, backlog: number, callback?: Function): Server;
+    listen(path: string, callback?: Function): Server;
+    listen(handle: Object, callback?: Function): Server;
+    close(callback?: Function): Server;
+    address(): { port: number; fmaily: string; address: string; };
```

Comment:
I don't see this in [http.Server](https://nodejs.org/api/http.html#http_class_http_server) but I do see this in [net.Server](https://nodejs.org/api/net.html#net_server_address). Am I looking at the wrong docs?